Bugzilla/BugUrl/GitLab.pm

```diff
@@ -20,9 +20,10 @@ use base qw(Bugzilla::BugUrl);
 sub should_handle {
   my ($class, $uri) = @_;
 
-# GitLab issue URLs can have the form:
+# GitLab issue and merge request URLs can have the form:
 # https://gitlab.com/projectA/subprojectB/subprojectC/../issues/53
```

Comment:
This link is for issues only.
Should we change this to `../[issues|merge_requests]/53` ?